app/models/concerns/votes_counter.rb

```diff
@@ -3,11 +3,17 @@ module VotesCounter
 
   included do
     scope :with_votes, -> {
-      joins("LEFT JOIN votes ON votes.voteable_id = #{table_name}.id AND votes.voteable_type = '#{to_s}'").select("#{table_name}.*, SUM(votes.value) AS total_votes").group("#{table_name}.id")
+      joins("LEFT JOIN votes ON votes.voteable_id = #{table_name}.id AND " \
```

Comment:
What do you think of using `AREL` here instead of raw sql. The reason why sql should be only used as a last resort is that the `SQL` of different sql databases varies. It will be very difficult to switch to a different database engine when you have a lot of sql queries scattered throughout your application. 

Check out [this blog post](https://www.viget.com/articles/composable-sql-queries-in-rails-using-arel) to learn how powerful  `AREL` can be. You may never need to write a single line of SQL again. Unless the query is just too complex.